test/e2e/standalone-html/index.html

```diff
@@ -9,13 +9,16 @@
   <body>
     <div id="app"></div>
     <script src="/dist/bundles/elastic-apm-js-base.umd.min.js"></script>
+    <script src="/dist/bundles/elastic-apm-opentracing.umd.min.js"></script>
```

Comment:
are we planning to release opentracing support as separate script instead?

Reply:
I think that's a better approach, considering the feature might not be used in a lot of use-cases. I would prefer to keep our main bundle size small.

What do you think?